src/components/PopoverMenu.vue

```diff
@@ -1,4 +1,5 @@
 <script setup>
+import { defineProps } from "vue";
```

Comment:
defineProps is a compiler macro, importing it creates a warning when running `yarn dev`

Reply:
fixed